tests/ui/stub-attribute/attribute.rs

```diff
@@ -1,6 +1,6 @@
 // Copyright Kani Contributors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
-//
+// kani-flags: -Z stubbing
```

Comment:
Do we want to make a copy of this test, use `--enable-stubbing` and just check that the warning is there?

I know this goes against @tautschnig advice of removing all uses of `--enable-stubbing`. But if we have an option, even a deprecated one, I don't think it's a good idea to leave it untested.

Reply:
I agree with you. @tautschnig would that be OK?

Reply:
If the test is all about testing the warning, then, sure, I have no objection at all! My concern was solely about leaving deprecated options in there that will continue to work for a while, and then (once we actually remove `--enable-stubbing`) having a very large cleanup commit.